pkg/controller/nodenetworkconfigurationpolicy/policyconditions/conditions.go

```diff
@@ -93,31 +93,33 @@ func Update(cli client.Client, policy *nmstatev1alpha1.NodeNetworkConfigurationP
 		}
 
 		nodes := corev1.NodeList{}
-		nodeSelectorFilter := client.MatchingLabels(policy.Spec.NodeSelector)
-		err = cli.List(context.TODO(), &nodes, nodeSelectorFilter)
+		err = cli.List(context.TODO(), &nodes)
 		if err != nil {
 			return errors.Wrap(err, "getting nodes failed")
 		}
-		numberOfNodesAffected := len(nodes.Items)
+
+		numberOfNodes := len(nodes.Items)
+		numberOfEnactments := len(enactments.Items)
 
 		// Let's get conditions with true status count
 		enactmentsCount := enactmentconditions.Count(enactments)
 
-		logger.Info(fmt.Sprintf("enactments count: %s", enactmentsCount.String()))
-
-		if numberOfNodesAffected == 0 {
-			message := "Policy does not match any node"
-			setPolicyNotMatching(&policy.Status.Conditions, message)
-		} else if enactmentsCount.Failed()+enactmentsCount.Available() == numberOfNodesAffected {
-			if enactmentsCount.Failed() > 0 {
-				message := fmt.Sprintf("%d/%d nodes failed to configure", enactmentsCount.Failed(), numberOfNodesAffected)
+		logger.Info(fmt.Sprintf("enactments count: %s", enactmentsCount))
+		if numberOfEnactments < numberOfNodes {
+			setPolicyProgressing(&policy.Status.Conditions, fmt.Sprintf("Policy is progresssing %d/%d enactments not ready", numberOfNodes-numberOfEnactments, numberOfNodes))
```

Comment:
Typically, you wait until the progress bar reaches 10/10. Let's make this "X/Y enactments finished".

Reply:
is not finishe is created, I will change to that.

Reply:
If you follow the comment above about change of conditions, it becomes "Finished", aka all NNCEs are either Degraded or Available. No?